mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java

```diff
@@ -168,8 +169,7 @@ public long highestModSeq(MailboxSession mailboxSession, MailboxId mailboxId) th
             update.bind()
                 .setUUID(MAILBOX_ID, mailboxId.asUuid())
                 .setLong(NEXT_MODSEQ, nextModSeq.getValue())
-                .setLong(MOD_SEQ_CONDITION, modSeq.getValue())
-                .setConsistencyLevel(ConsistencyLevel.SERIAL))
+                .setLong(MOD_SEQ_CONDITION, modSeq.getValue()))
```

Comment:
Same question

Reply:
it's actually a removal of the previous accidental insertion